src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DataMaskingTests.cs

```diff
@@ -21,49 +21,49 @@ namespace Microsoft.Azure.Commands.Sql.Test.ScenarioTests
     public class DataMaskingTests : SqlTestsBase
     {
 
-        [Fact(Skip = "test takes more than 5 sec")]
+        [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestDatabaseDataMaskingPolicyEnablementToggling()
```

Comment:
These tests were disabled because they were taking over 5 sec per tests. Based on the last test run they still take 760 seconds all together (and about 100 sec each). Please either fix the performance or exclude them from the CI test run. If you mark them as `[Trait(Category.AcceptanceType, Category.BVT)]` they will be run post PR merge in a separate job.